saltedge-compliance-connector-java-v2/saltedge-connector-sdk/src/main/java/com/saltedge/connector/sdk/callback/Config.java

```diff
@@ -0,0 +1,14 @@
+package com.saltedge.connector.sdk.callback;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class Config {
```

Comment:
I would give this a more specific name. Probably something like CallbackRestClientConfig. It might also make sense to move it into a config package as well.